src/main/resources/db/migration/V1_44__create_holding_details_table.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE holding_details
+(
+    id                      SERIAL PRIMARY KEY NOT NULL,
+    _id                     VARCHAR(255) NOT NULL,
+    _external_id            VARCHAR(255) NOT NULL,
+    symbol                  VARCHAR(255) NOT NULL,
+    country_id              VARCHAR(255) NOT NULL,
+    cusip                   VARCHAR(255) NOT NULL,
+    currency_id             VARCHAR(255) NOT NULL,
+    security_name           VARCHAR(255) NOT NULL,
+    legal_type              VARCHAR(7)  NOT NULL,
+    weighting               DECIMAL(10, 5) DEFAULT 0,
+    number_of_share         BIGINT DEFAULT 0,
+    share_change            BIGINT DEFAULT 0,
+    market_value            BIGINT DEFAULT 0,
+    sector                  INTEGER DEFAULT 0,
+    holding_ytd_return      DECIMAL(10, 5) DEFAULT 0,
+    region                  INTEGER DEFAULT 0,
```

Comment:
We should persist the region as enum instead of an integer.
i.e.
```
1 = UNITED_STATES
2 = CANADA
3	= LATIN_AMERICA

...

16 = NOT_CLASSIFIED
```